html/demos/janus.js

```diff
@@ -3045,7 +3045,7 @@ function Janus(gatewayCallbacks) {
 									return;
 								let inStats = false;
 								// Check if these are statistics on incoming media
-								if((res.mediaType === "video" || res.id.toLowerCase().indexOf("video") > -1) &&
+								if((res.kind === "video" || res.id.toLowerCase().indexOf("video") > -1) &&
```

Comment:
Wouldn't it better to keep both? Like

    res.mediaType === "video" || res.kind === "video" || ...

I just want to avoid scenarios where there's still older browsers that don't support the new syntax.

Reply:
Seems reasonable, I hope it won't be redundant as it was the only "mediaType" property used in the file.